packages/workbox-webpack-plugin/src/lib/generate-or-copy-sw.js

```diff
@@ -31,8 +31,7 @@ function sanitizeConfig(config) {
   const propertiesToRemove = [
     'chunks',
     'excludeChunks',
-    'filename',
-    'manifestFilename',
```

Comment:
Are these completely removed as options or is it just moved logic wise to somewhere else?

Reply:
They're removed (and hardcoded for the time being; some of this will be revised when we support `GenerateSW` vs. `InjectManifest`).